priority/management/commands/calculate-priority.py

```diff
@@ -0,0 +1,121 @@
+from django.core.management.base import BaseCommand, CommandError
+from sqlalchemy import create_engine, Table
+import numpy as np
+import pandas as pd
+from datetime import date, timedelta
+from project.models import Project
+from funding.models import Attribution
+from priority.models import slurm_priority
+import sys
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        #number of QOS levels
+        QOS_levels=4.0
+        Today=date.today()
+        #engine for reading/writing sql files
+        # sqlite://<nohostname>/<pathtofile>
+        engine = create_engine('sqlite:///db.sqlite3', echo=False)
+        df = pd.read_csv ('~/Documents/full_sacct_dump.csv',sep='|')
+        #df = pd.read_csv ('~/Documents/Cogs3/cogs3/priority/fixtures/TEST_dump.csv',sep='|')
+        #remove NANs
+        df=df.dropna(how='any')
+        #convert cpu time from core seconds to core hours
+        df.loc['CPUTimeRAW']=df['CPUTimeRAW'].div(3600)
+        #df['CPU-HOURS']=cpuhours.astype(int)
+        #seperate into gpu and not gpu
+        gpu=df[(df.Partition=="gpu")|(df.Partition=="xgpu")]
+        compute=df[(df.Partition!="gpu")|(df.Partition!="xgpu")]
+        compute.to_csv(path_or_buf='~/Documents/Cogs3/cogs3/compute.csv')
+        #calculate the total raw cpu time used for each project
+        cpu_sum=compute.groupby('Account')['CPUTimeRAW'].sum().astype(int).to_frame(name='cpu_sum')
+        gpu_sum=gpu.groupby('Account')['CPUTimeRAW'].sum().astype(int).to_frame(name='gpu_sum')
+        print(gpu_sum)
+        #generate random ap values for testing
+        #default_Ap=pd.Series(np.random.randint(50000,250000,len(SCW['Ap'])),name='Ap').astype(int)
+        SCW=caculate_AP()
+        #combine the three data frames into one
+        SCW = pd.merge(SCW,cpu_sum, on='Account', how='outer').fillna(0)
+        SCW = pd.merge(SCW,gpu_sum, on='Account', how='outer').fillna(0)
+        #give all new entries an Ap of 50000
+        #SCW.update(default_Ap,overwrite=False,filter_func=new_data)
+        SCW.Ap.loc[SCW.Ap==0]=50000
```

Comment:
This logic is already handled in `caculate_AP` - all projects start at 50k there, so no need to duplicate this logic here.